tests/integration/end2end/bert.yaml

```diff
@@ -0,0 +1,36 @@
+!Experiment
+
+name: bert
+
+pipeline:
+  dataset: !TabularDataset.from_path
+    train_path: {top_level}/tests/data/dummy_tabular/train.csv
+    val_path: {top_level}/tests/data/dummy_tabular/val.csv
+    sep: ','
+    transform:
+      text: !BertTextField
+        alias: 'bert-base-uncased'
+      label: !LabelField
+
+  # teacher: !TextClassifier
```

Comment:
The `teacher` and `finetune` are commented because of memory issues. Locally it is working.